src/ExpressiveInstaller/Resources/src/App/Repository/PostRepository.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Post;
+use Kocal\Expressive\Database\Doctrine\DoctrineRepository;
```

Comment:
Why extend Kocal\Expressive\Database\Doctrine\DoctrineRepository? It's a pretty personal package. What if I want have my own way of doing things or don't want the basic Doctrine class?